src/parser/formatting-evaluator-module.ts

```diff
@@ -84,6 +72,35 @@ export class FormattingEvaluatorModule implements Module {
     return result;
   }
 
+  private _calculateFormattingTotal(
+    formatting: ReturnType<typeof this._getFormattingScore>["formatting"],
+    multiplierFactor: Multiplier
+  ): Decimal {
+    if (!formatting) return new Decimal(0);
+
+    return Object.values(formatting)
+      .reduce((acc, curr) => {
+        let sum = new Decimal(0);
+
+        for (const symbol of Object.keys(curr.symbols)) {
+          const count = new Decimal(curr.symbols[symbol].count);
+          const symbolMultiplier = new Decimal(curr.symbols[symbol].multiplier);
+          const formattingElementScore = new Decimal(curr.score);
+          const exponent = this._wordCountExponent;
+
+          sum = sum.add(
+            count
+              .pow(exponent) // (count^exponent)
+              .mul(symbolMultiplier) // symbol multiplier
+              .mul(formattingElementScore) // comment type multiplier
+              .mul(multiplierFactor.multiplier) // formatting element score
+          );
```

Comment:
word count exponent is being applied to all symbols, either we leave it like that but we should rename it to symbol exponent, or we can limit it to only words @0x4007

Reply:
I'm not quite sure but I feel like word count specifically makes the most sense.

Reply:
The problem is that config doesn't define word multiplier anymore, instead symbol regexes are used and technically two regexes which are a little bit different can both represent words so it's hard to apply the exponent only to the word count

Reply:
I had a proposal for "segments" which would be an alias for word regex, sentence regex, paragraph regex, entire comment regex. Perhaps we can rely on that as the main "word counter"

Reply:
I agree with having segments, we can keep regexes but I don't think many partners will bother to use that